common/pom.xml

```diff
@@ -94,5 +94,24 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
         </dependency>
+        <!--product metrics dashboard dependencies-->
+        <dependency>
+            <groupId>org.influxdb</groupId>
+            <artifactId>influxdb-java</artifactId>
+            <version>2.5</version>
```

Comment:
Please define the versions on root pom and refer them in modules. (You can refer other dependencies to see how its done)

Reply:
+1